packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTableDepositRow.tsx

```diff
@@ -108,13 +108,14 @@ function DepositRowTime({ tx }: { tx: MergedTransaction }) {
 
   return (
     <div className="flex flex-col space-y-3">
-      <Tooltip content={<span>L1 Transaction Time</span>}>
-        <TransactionDateTime standardizedDate={tx.createdAt} />
-      </Tooltip>
-      {tx.resolvedAt && (
+      {tx.resolvedAt ? (
```

Comment:
This condition will only show 1 date at a time no matter what the deposit status is. If the transaction genuinely succeeded it will have 2 timestamps - created and resolved and we would want to show both in that case. 
I think the condition we require here should be along these lines-
`tx.resolvedAt && tx.createdAt!==tx.resolvedAt && <Resolved date..>`
`tx.createdAt && <Creation date..>`